### PR TITLE
Fix: Make tooltips keyboard accessible (Fixes #33)

### LINKED ISSUE #33: Make tooltips keyboard accessible
The pageNav button tooltips only appear on mouseover, which means they fail WCAG 2.1.1. 

Per [the APG Tooltip pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/), the tooltips should also appear on focus, and disappear on blur or <kbd>esc</kbd>.

### PR DESCRIPTION
**Pull request by hanshillen**
Fixes #33

### Fix

* Added focus, blur, and keyup event handlers to page nav buttons for showing / hiding tooltip.

Reference: https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/

### CODE REVIEW

File: js/view.js [hunk 1]
```diff
@@ -136,8 +139,23 @@ define([
           break;
       }
     },
+    onBlur: function(event) {
+      Adapt.trigger('tooltip:remove');
+    }
```
Comment (oliverfoster) [inline js/view.js:144]: missing some commas

File: js/view.js [hunk 2]
```diff
@@ -136,8 +139,23 @@ define([
           break;
       }
     },
+    onBlur: function(event) {
```
Comment (oliverfoster) [inline js/view.js:142]: missing whitespace

File: js/view.js (hunk 1 above)
Comment (hanshillen) [reply js/view.js]: Sorry about that, copied this from a class while rushing. Fixed now

File: js/view.js (hunk 2 above)
Comment (hanshillen) [reply js/view.js:143]: Fixed

**Pull request merged**